### PR TITLE
TMEDIA 734: Add slides for autoplay and go to next slide like autoplay

### PR DESCRIPTION
**Pull request by JackHowa**
- Autoplay works the same way as next and before should
- Fixes subtask as well https://arcpublishing.atlassian.net/browse/TMEDIA-945
# Before 

https://user-images.githubusercontent.com/5950956/187993917-088791ad-c535-4d84-b1a8-be23a6800b12.mov

# After 

https://user-images.githubusercontent.com/5950956/187993907-68138a8c-6ebc-4ed5-8fd4-376df3de82e7.mov

### CODE REVIEW

**Comment by JackHowa** [comment]
Might be seeing a regression http://localhost:60003/?path=/story/components-carousel--display-autoplay-toggle-with-multiple-slides-showing-full-screen-toggle-showing-show-label

File: src/components/carousel/index.jsx [hunk 1]
```diff
@@ -231,7 +231,9 @@ const Carousel = ({
 			setIsAutoplaying(false);
 		} else {
 			setSlide(slide + 1);
-			setPosition(position - 100 / slidesToShowInView);
+			const slideOffset =
+				carouselElement.current.querySelector(".c-carousel__slide:nth-of-type(2)")?.offsetLeft || 0;
```
Comment (nschubach) [inline src/components/carousel/index.jsx:235]: I both don't like the querySelector (should use references in React) and I am curious about the "2" here.  Is this ONLY ever the second item?

Comment (JackHowa) [reply src/components/carousel/index.jsx:235]: this is how it's done for the next and previous logic. the current logic is not working. 

We are using a reference with carouselElement. `nth-of-type` is for calculating how much to slide

Comment (nschubach) [reply src/components/carousel/index.jsx:235]: Oh, I see, it's just grabbing the second one and using it's left position to determine the shifting value.

I can accept this, but as a grander issue, I think I'll note this as a wiggle week change or something.  It's probably not breaking, but more of an implementation detail.

**Review by nschubach — APPROVED** [review]

**Pull request merged**